test/util.ts

```diff
@@ -24,7 +24,7 @@ export async function clearExistingKitConfigurationFile() {
 }
 
 export function getExtension() {
-    const cmt = vscode.extensions.getExtension<CMakeTools>('ms-vscode.cmake-tools');
+    const cmt = vscode.extensions.getExtension<CMakeProject>('ms-vscode.cmake-tools');
```

Comment:
This should probably return the API, but that's currently disabled.  Maybe put a comment here to remind us.

Reply:
Copied som parts of the other util.thisExtension function for this function.